client/my-sites/hosting/web-server-settings-card/index.js

```diff
@@ -221,32 +294,43 @@ const WebServerSettingsCard = ( {
 
 	return (
 		<Card className="web-server-settings-card">
+			{ config.isEnabled( 'hosting/datacenter-picker' ) && (
+				<QuerySiteGeoAffinity siteId={ siteId } />
+			) }
 			<QuerySitePhpVersion siteId={ siteId } />
 			<QuerySiteStaticFile404 siteId={ siteId } />
 			<MaterialIcon icon="build" size={ 32 } />
 			<CardHeading>{ translate( 'Web Server Settings' ) }</CardHeading>
-			<p>For sites with specialized needs, fine-tune how the web server runs your website.</p>
+			<p>
+				{ translate(
+					'For sites with specialized needs, fine-tune how the web server runs your website.'
+				) }
+			</p>
+			{ config.isEnabled( 'hosting/datacenter-picker' ) && getGeoAffinityContent() }
 			{ getPhpVersionContent() }
 			{ getStaticFile404Content() }
-			{ ( isGettingPhpVersion || isGettingStaticFile404 ) && <Spinner /> }
+			{ ( isGettingGeoAffinity || isGettingPhpVersion || isGettingStaticFile404 ) && <Spinner /> }
```

Comment:
The `<Spinner />` is always showing for older sites, because `geoAffinity` is `null` and `isGettingGeoAffinity` will be `true`.

Line 325:
```
isGettingGeoAffinity: ! props.disabled && ! geoAffinity,
```

Reply:
@sejas Thanks! Fixed this up with ba6e71ca93de235ca60308b56776b1fbaf2c9813